Functions.Rmd

```diff
@@ -683,9 +683,9 @@ str(h09(10))
 args(sample)
 ```
  
-It looks like both `x` and `size` are required, but in fact `sample()` uses `missing()` to provide a default for `size` if it's not supplied. If I was to rewrite sample myself[^sample], I'd use an explicit `NULL` to indicate that `size` can be supplied, but it's not required:
+It looks like both `x` and `size` are required, but in fact `sample()` uses `missing()` to provide a default for `size` if it's not supplied. If I were to rewrite sample myself[^sample], I'd use an explicit `NULL` to indicate that `size` is not required but can be supplied:
 
-[^sample]: Note that this only implements one way of calling `sample()`: you can also call it with a single integer, like `sample(10)`. This unfortunately makes `sample()` prone to silent errors in situations like `sample(x[i])`.
+[^sample]: Note that this only one way to implement a call to `sample()`: you can also call it by using a single integer, like `sample(10)`. This unfortunately makes `sample()` prone to silent errors in situations like `sample(x[i])`
```

Comment:
I think this loses the original intent; i.e. there are multiple ways to call `base::sample()`, and I only implement one.

Reply:
That's not quite right either. Maybe we should just kill this footnote altogether? It's not that important a point here.

Reply:
ok. will drop it.